eslintrc.json

```diff
@@ -2,114 +2,115 @@
   "extends": "standard",
   "rules": {
     "array-bracket-spacing": [
-      2,
+      "error",
       "never",
       {
       }
     ],
     "brace-style": [
-      2,
+      "error",
       "1tbs",
       {
         "allowSingleLine": true
       }
     ],
     "camelcase": [
-      2,
+      "error",
       {
         "properties": "never"
       }
     ],
     "comma-dangle": [
-      2,
+      "error",
       "never"
     ],
     "comma-style": [
-      2,
+      "error",
       "last"
     ],
     "curly": [
-      2,
+      "error",
       "all"
     ],
-    "dot-notation": 2,
-    "eol-last": 2,
-    "eqeqeq": 2,
-    "guard-for-in": 2,
+    "dot-notation": "error",
+    "eol-last": "error",
+    "eqeqeq": "error",
+    "guard-for-in": "error",
     "indent": [
-      2,
+      "error",
       2,
       {
         "SwitchCase": 1
```

Comment:
I'd like to also enable VariableDeclarator, ArrayExpression and ObjectExpression added here.

The current indent block is actually duplicated from [standard](https://github.com/feross/eslint-config-standard/blob/master/eslintrc.json#L52) 

see http://eslint.org/docs/rules/indent